DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/LearningLog.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateProgress
+@model LearningLogViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/learningLog.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Delegate learning log";
+  var routeParamsForBackLink = new Dictionary<string, string?> {
+    {
+      "accessedVia", Model.AccessedVia.Name
+    }, {
+      "progressId", Model.ProgressId.ToString()
+    },
+  };

```

Comment:
I assume this is something the auto formatter is doing?

Reply:
Afraid so. I'm not sure this page will get any further changes to it, so could manually reformat it to something nicer?

Reply:
Probably not worth it, easier to just leave it how the auto format will do it and cause less hassle in the future